### PR TITLE
Automatic backups

### PR DESCRIPTION
**Pull request by erikdesjardins**
Tested in browser: Chrome 60, Firefox 54, Edge 15

re #4266

### CODE REVIEW

File: locales/locales/en.json [hunk 1]
```diff
@@ -1632,6 +1632,33 @@
...
+		"message": "Open a prompt to reload other tabs."
+	},
+	"backupAndRestoreReloadWarningAuto": {
+		"message": "Automatically reload other tabs."
+	},
+	"backupAndRestoreAutomaticBackupsTitle": {
+		"message": "Automatic Backups"
+	},
+	"backupAndRestoreAutomaticBackupsDesc": {
+		"message": "WARNING: THIS OPTION IS EXPERIMENTAL AND MAY DELETE ALL OF YOUR SETTINGS, TAGS, ETC. AT ANY TIME.\n\nSync your backups to the specified provider.\n\nOn pageload, every five minutes, if a new backup is detected, it will be imported, replacing all settings. Otherwise, a new backup will be created. Changes aren't (and will not be) reconciled, so if you quickly switch between two computers you may lose some changes."
```
Comment (erikdesjardins) [inline locales/locales/en.json]: This ought to do it. Yes, definitely. 😉

**Comment by erikdesjardins** [comment]
@corylulu so I guess we can just mark it experimental, why not. ¯\\\_(ツ)_/¯

**Comment by corylulu** [comment]
@erikdesjardins Nice, might be a good idea to store an md5 of the data to help safe guard against a bad backup/restore if it's not already there.

**Comment by corylulu** [comment]
@erikdesjardins What do I need to do to make this work if I'm building it myself? Should I need a specific chrome.pem or anything while packing the extension or will I need to supply my own client ID? 

I can't get it to not show `Error in target's "authFlow" handler: Authorization page could not be loaded.`

**Pull request merged**